lib/geordi/commands/chromedriver_update.rb

```diff
@@ -6,8 +6,11 @@
 installed Chrome.
 LONGDESC
 
+option :show_warning, type: :boolean, default: true,
+  desc: 'Show a warning if the chrome and chromedriver versions are already the same and nothing needs to be done.'
```

Comment:
I have objections regarding the option naming. "show warning" could be about anything. Let's get a bit closer to the actual thing we want here. What about `quiet_if_matching`? Default would be false, and the "no update" note would not be printed if the option is true.

Please also shorten the option description to about half its current length. This keeps it from wrapping when printed on the bash (confer `geordi help cucumber` > debug option description).